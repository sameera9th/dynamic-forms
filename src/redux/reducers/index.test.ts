import rootReducer, { ReduxStateTypes } from './index';

describe('root reducer', () => {
    it('should return the rootReducer', () => {
        expect(rootReducer({} as ReduxStateTypes, { type: undefined })).toMatchSnapshot();
    });
});
