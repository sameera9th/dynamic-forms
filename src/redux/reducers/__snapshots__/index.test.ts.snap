// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`root reducer should return the rootReducer 1`] = `
Object {
  "formFields": Object {
    "JSONVisibility": false,
    "data": Array [],
    "error": "",
    "fetching": false,
    "fieldsJSON": Object {},
  },
}
`;
